src/registrar/templates/domain_delete.html

```diff
@@ -0,0 +1,128 @@
+    <p>If you would like to delete this domain, you can request that it be deleted at any time. 
+      Check the box at the bottom to indicate you understand that your domain will be deleted 7 days after the request.
+      <ul class="usa-list">
+      <li>Once you request to delete this domain, it will be placed "on hold" in the .gov registry for 7 days before deletion.
+        While “on hold,” the domain will be administratively paused, so it can’t be edited and won’t resolve in DNS.
+      </li>
+      <li>When your domain is deleted, it will be removed from the .gov registry. 
+        This means that websites and emails associated with the domain will no longer work.
+      </li>
+      <li>After the domain is deleted from the .gov registry, it cannot be recovered.
+        To obtain the same domain name, a domain manager will have to make a new request for the domain.
+      </li>
+        <li>A domain manager can cancel the deletion request by email <a href='mailto:help@get.gov' class='usa-link'>help@get.gov</a>.
+      </li>
+      </ul>
+    </p>
+
+
+      <fieldset class="usa-fieldset">
+        <legend>
+        <h2 class="summary-item__title 
+          text-primary-dark 
+          text-semibold 
+          margin-top-0 
+          margin-bottom-05 
+          padding-right-1">
+        Acknowledgement </h2>          
+        </legend>
+    
+        <form method="post" action="{% url 'domain-delete' domain_pk=domain.id %}" id="submit-domain-deletion-form"> 
+          {% csrf_token %}
+          <div class="usa-checkbox">
+
+            {% if form.is_policy_acknowledged.errors %}
+            {% for error in form.is_policy_acknowledged.errors %}
+            <div class="usa-error-message display-flex" role="alert">
+              <svg class="usa-icon usa-icon--large" focusable="true" role="img" aria-label="Error: Check the box if you understand that your domain will be deleted within 7 days of making this request.">
+                <use xlink:href="{%static 'img/sprite.svg'%}#error"></use>
+              </svg>
+              <span class="margin-left-05">{{ error }}</span>
+            </div>
+            {% endfor %}
+        {% endif %}
+
+          <input 
+            class="usa-checkbox__input" 
+            id="delete-domain-checkbox" 
+            type="checkbox" 
+            name="is_policy_acknowledged"
+            value="True"
+            {% if form.is_policy_acknowledged.value %}checked{% endif %}
+            >
+            <label class="usa-checkbox__label" for="delete-domain-checkbox">
+              I understand that my domain will be deleted within 7 days of my request for deletion. After that, it cannot be recovered. <abbr class="usa-hint usa-hint--required" title="required">*</abbr>
+            </label>
+          </div>
+
+          <button
+          type="button" 
+          id="request-domain-deletion-button"
+          class="usa-button usa-button--outline usa-button--outline-red margin-top-3"
```

Comment:
```suggestion
          class="usa-button usa-button--outline usa-button--outline-secondary margin-top-3"
```
^ (see comment about button styling and reusing existing declarations)